### ISSUE TITLE
How are subgroups to be dealt with in derivatives? E.g. patients versus controls. Perhaps like this:

### ISSUE THREAD

**Issue by chrisfilo** [bids-standard/bids-specification#163]
How are subgroups to be dealt with? E.g. patients versus controls. Perhaps like this:

    group-patients
    group-controls

where all subjections would be

    group-all

_Originally posted by @robertoostenveld in https://github.com/_render_node/MDExOlB1bGxSZXF1ZXN0MjM4OTg3Njgy/timeline/more_items_

**Comment by oesteban** [comment]
Back in september I added some about this in the Google Docs, but it seems it disappeared - https://github.com/bids-standard/bids-specification/pull/109#discussion_r261477328

**Comment by chrisfilo** [comment]
Indeed handling group results have been removed from https://github.com/bids-standard/bids-specification/pull/109 to make the scope more manageable. Sorry this wasn't clear. This issue is to track this feature which hopefully can be covered by a new PR after #109 is merged.

**Comment by oesteban** [comment]
Yep, I think it disappeared before you even started the daunting task of #109. Anyways, that is irrelevant now, let's see whether we can build a neat implementation of groups :)

**Comment by TheChymera** [comment]
What about putting this information in a project-wide metadata file?
That would help make analysis blind, since researchers could keep the file unopened (or unwriten) up until the conclusion of the analysis. If groups are separated by directories or file names, it's going to be hard to even perform basic preprocessing QC in an unbiased fashion.

**Comment by rob-luke** [comment]
Has this been addressed via the example participant.tsv file where there is a group column?

**Comment by CPernet** [comment]
I'd go for a simple solution - a recommended column in participants.tsv and possibly part of the name sub-control001, sub-patient001, ect ...  I used that before, worked great

Having recommended naming would however help for pipelines (control is not the same as healthy, for instance you can have a patient group of interest vs a control patient group)

**Comment by oesteban** [comment]
> What about putting this information in a project-wide metadata file?

This is readily available with BIDS "raw".

> That would help make analysis blind, since researchers could keep the file unopened (or unwriten) up until the conclusion of the analysis. If groups are separated by directories or file names, it's going to be hard to even perform basic preprocessing QC in an unbiased fashion.

This is not a use case of this PR - albeit this is a good practice, BIDS (incl. Derivatives) already permits this be done easily. It's a choice of the researcher to encode things without obvious identifiers.

This PR is for encoding results that are group-wise in nature. Such as the custom template above.

> a recommended column in participants.tsv

This is what @TheChymera recommended above, and perfectly doable ATM.

> possibly part of the name sub-control001, sub-patient001, ect ... I used that before, worked great

Encoding variables within entity values is discouraged and definitely a bad idea. Best example is QC as Chris was mentioning.

That said, this suggestion does not solve the problem that analysis generates some interim results that derive from two or more different participants.

**Comment by omar-rifai** [comment]
Sorry for reviving an old post but I was wondering whether there was a followup on this discussion with regards to templates and other grouped intermediate results. I can not seem to find a clear consensus on how to proceed in this case.

**Comment by yarikoptic** [comment]
I think recently introduced [cohort](https://bids-specification.readthedocs.io/en/latest/appendices/entities.html#cohort) entity is the answer here, and examples of use: https://bids-specification.readthedocs.io/en/stable/derivatives/atlas.html#example-multi-cohort-template.

NB  If only it came with `cohorts.tsv`! (ref: https://github.com/bids-standard/bids-specification/issues/2283) to describe them.
